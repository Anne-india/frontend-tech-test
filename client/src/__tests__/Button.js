import React from 'react';
import Button from '../components/Button';
import renderer from 'react-test-renderer';

describe('Button Component', () => {
  let props;

  beforeEach(() => {
    props = {
      type: 'button',
      className: "abc",
      text: 'test'
    };
  });

  it('renders without crashing', () => {
    const tree = renderer.create(<Button {...props} />)

    expect(tree.toJSON()).toMatchSnapshot();
  });
});