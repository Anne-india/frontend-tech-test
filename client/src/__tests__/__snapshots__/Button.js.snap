// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button Component renders without crashing 1`] = `
<button
  className="abc"
  type="button"
>
  test
</button>
`;
